src/style/main.scss

```diff
@@ -100,6 +100,7 @@ main {
             transition: background-color 300ms ease, border 300ms ease;
         }
 
+        select,
```

Comment:
Add this selector to the block above, otherwise, the dropdown doesn't have rounded corners and a proper font size

Reply:
Resolved in 4d36c872caa1998c5f4af57e315dc5b0006b88b4